src/bundle/Resources/public/js/scripts/helpers/notification.helper.js

```diff
@@ -55,31 +55,27 @@
         });
 
     /**
-     * Dispatches danger notification event
+     * Dispatches error notification event
      *
-     * @function showDangerNotification
-     * @param {String} message
+     * @function showErrorNotification
+     * @param {(string|Error)} error
      */
-    const showDangerNotification = (message) =>
+    const showErrorNotification = (error) => {
+        const isErrorObj = error instanceof Error;
+        const message = isErrorObj ? error.message : error;
+
         showNotification({
             message,
-            label: NOTIFICATION_DANGER_LABEL,
+            label: NOTIFICATION_ERROR_LABEL,
         });
-
-    /**
-     * Dispatches danger notification event
-     *
-     * @function showErrorNotification
-     * @param {Error} error
-     */
-    const showErrorNotification = (error) => showDangerNotification(error.message);
+    };
 
     eZ.helpers = eZ.helpers || {};
     eZ.helpers.notification = {
+        showNotification,
         showInfoNotification,
         showSuccessNotification,
         showWarningNotification,
-        showDangerNotification,
```

Comment:
We cannot just remove this because of BC. We should add deprecation warning and remove this in 2.0 of admin-ui

Reply:
We should always use the version of the package. 
`ezplatform-admin-ui` is now 1.x so next version for removing deprecations is 2.0 (which most likely will be released as eZPlatform 3.0).